next.config.js

```diff
@@ -7,16 +7,14 @@ const nextConfig = {
       {
         source: '/',
         destination: '/dashboard',
-        permanent: true,
+        permanent: false,
```

Comment:
unrelated to custom components, just fixing an issue I discovered; by using `permalink: true`, we set the redirect permanently (big surprise), meaning that if you run `localhost:3000/` in the future it will always redirect to `localhost:3000/dashboard` (even if running a separate app entirely). using `permalink: false` still gets the job done